tracking_manager/models/ir_model.py

```diff
@@ -159,7 +158,6 @@ class IrModelFields(models.Model):
     )
     trackable = fields.Boolean(
         compute="_compute_trackable",
-        store=True,
```

Comment:
Why did you remove store=True?

I think that you may need to add something in the depends of your compute if the field isn't (re-)compute correctly.

Reply:
The trackable attribute is already computed, but with this PR we change the implementation how it is computed. 
Changing the implementation does not change the stored value automatically.
I agree another options would be an migration update script to recalculate all values. 

In think this is the only place where trackable is used. I am not sure if this syntax requires a stored field. But this could also be replaced by an lambda. Beside not changing it, I do not see the benefit of storing this simple value. 

https://github.com/OCA/server-tools/blob/485de9f8f8bba0df1fb905a47bc99e5cccb434a3/tracking_manager/models/ir_model.py#L128-L135

Reply:
I think that a migration script might be needed like you suggested but I am not 100% sure about that..

Reply:
Maybe there is a 3rd opinion over the weekend. I am personally not so a big fan of store=True on computed values. But indeed, changing it has also its risks.

Reply:
I ll prefer to keep it stored because in some cases, we want to save a custom choice of fields tracked, and this independently to its kind (read only, related, ...).

Reply:
I think "trackable" is only used as a filter when showing all trackable fields in the list. If this value is False, the field is not in the list and you cannot activating the track. There is also no possibility to change this value in the UI.
![image](https://github.com/OCA/server-tools/assets/1799080/75e5958e-eb50-4ca9-b907-a2eef96ee662)

I do not see why this field needs to be persistent, but I agree changing it could also have unwanted side effects. But I did not have time yet to write the necessary upgrade script, to recalculate all values with the update. Without an update of the stored values, removing the readonly filter has no effect.